### PR TITLE
add Build Waiting State

### LINKED ISSUE #1562: Build Waiting State
As a user I'd like to see a cleaned up waiting state (the waiting state == the screen a user sees when they are waiting for their ethereum tx to clear)

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- Above design is implemented and works on all scenarios that require this screen.
- The screen is responsive
- Animate rocket ship going up slowly and possibly looping (up to animation experimentatation) and sparks spinning.

1 copy 21

### PR DESCRIPTION
**Pull request by iamonuwa**
Fixes #1562

### CODE REVIEW

File: app/dashboard/templates/bounty_details.html [hunk 1]
```diff
@@ -53,7 +55,7 @@ <h6>{% trans "Be the OSS Funding you wish to see in the world." %}</h6>
             <img src="{% static "v2/images/quotes.png" %}" style="float: left;" />
             <div class="quote_text"></div>
             <div class="quote_author"></div>
-            <input id="quoteRefresh" class="button button--primary" name="refresh" type='submit' onclick='javascript:waitingRoomEntertainment()' value="Refresh Quote" >
+            <!-- <input id="quoteRefresh" class="button button--primary" name="refresh" type='submit' onclick='javascript:waitingRoomEntertainment()' value="Refresh Quote" > -->
```
Comment (opay-perfect) [inline app/dashboard/templates/bounty_details.html]: Why not just delete this instead of commenting it out?

**Comment by thelostone-mc** [comment]
finally a PR from @iamonuwa :D 

(I know this is not ready for review but just a heads up -> you'd want to shift away from inline css before you submit for review and try and stick to `rem` as against to `px` )

**Comment by iamonuwa** [comment]
@thelostone-mc noted. This is actually what I need to request for review

**Comment by owocki** [comment]
looks like the css needs some fixes, but that were headed in the right direction to me! [screenshot](http://bits.owocki.com/3A1B1J0E1u2J/Screen%20Shot%202018-07-03%20at%2012.58.42%20PM.png)

**Comment by thelostone-mc** [comment]
@iamonuwa gimme a heads up when this is ready for review

**Comment by iamonuwa** [comment]
Will be ready in an hour time

**Comment by iamonuwa** [comment]
@owocki image

File: app/dashboard/templates/bounty_details.html [hunk 2]
```diff
@@ -23,21 +23,7 @@
...
-        </div>
-        <div class="transaction-status" style="display:none;">
-          <div style='width:100%; font-size: 20px; color: #3d3d46;'>
-            {% trans "Your transaction is being posted to the Ethereum blockchain" %}
-            <br>
-            {% trans "Your page will automatically update when the information has been synced to Web3" %}
-          </div>
-        </div>
-
+        {% include 'shared/bounty_hero.html' %}
```
Comment (thelostone-mc) [inline app/dashboard/templates/bounty_details.html]: could we rename it to something like `bounty_waiting_state.html` or something better ?

File: app/dashboard/templates/bounty_details.html [hunk 3]
```diff
@@ -53,7 +39,7 @@ <h6>{% trans "Be the OSS Funding you wish to see in the world." %}</h6>
             <img src="{% static "v2/images/quotes.png" %}" style="float: left;" />
             <div class="quote_text"></div>
             <div class="quote_author"></div>
-            <input id="quoteRefresh" class="button button--primary" name="refresh" type='submit' onclick='javascript:waitingRoomEntertainment()' value="Refresh Quote" >
+            <!-- <input id="quoteRefresh" class="button button--primary" name="refresh" type='submit' onclick='javascript:waitingRoomEntertainment()' value="Refresh Quote" > -->
```
Comment (thelostone-mc) [inline app/dashboard/templates/bounty_details.html]: we can do away with code which isn't being used

File: app/dashboard/templates/bounty_details.html (hunk 2 above)
Comment (iamonuwa) [reply app/dashboard/templates/bounty_details.html]: Okay

**Comment by mbeacom** [comment]
@SaptakS Sure thing.  We can deploy this following the product sync.

File: app/dashboard/templates/shared/bounty_waiting_state.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see <http://www.gnu.org/licenses/>.
+{% endcomment %}
+{% load i18n static %}
+<div class="bg-container">
+  <div class="transation-header mt-5" style="display: none;">
+    <h2 class="my-2">Transaction Processing</h2>
```
Comment (mbeacom) [inline app/dashboard/templates/shared/bounty_waiting_state.html]: Can we wrap this in `trans` ?

**Comment by iamonuwa** [comment]
@PixelantDesign, am guessing you are using a 4k screen. I can add target that

**Comment by owocki** [comment]
i think we'll need to have this page working in mobile, tablet, desktop (4k or not)

**Comment by owocki** [comment]
this looks all screwed up on my local [screenshot](http://bits.owocki.com/3V1f20012z1P/Screen%20Shot%202018-07-05%20at%206.20.33%20PM.png)

**Comment by mbeacom** [comment]
local_waiting

**Comment by owocki** [comment]
still looks screwed up to me on google chrome / macbook pro at 100% zoom [screenshot](http://bits.owocki.com/3I2m2725210P/Screen%20Shot%202018-07-06%20at%202.14.41%20PM.png)

**Comment by owocki** [comment]
ok seems to be working to me now! http://bits.owocki.com/2S0h3Y2X3v37/Screen%20Shot%202018-07-08%20at%209.17.56%20PM.png

**Comment by owocki** [comment]
ok to 🚢 ?

**Pull request merged**